tests/js/web/Notifications/FaviconEditorTest.js

```diff
@@ -0,0 +1,115 @@
+const FaviconEditor = require('../../../../js/Notifications/FaviconEditor');
+const $ = require('jquery');
+const vanillaFavicon = require('./snapshots/favicon');
```

Comment:
Small thing, but this is more of a "fixture" than a snapshot as you wouldn't expect to need to update the fixture image very often, should this be in a folder called `fixtures` to differentiate them?

Reply:
Yep I agree, I'll change this.